crates/iota-sdk-types/src/crypto/zklogin.rs

```diff
@@ -522,7 +522,7 @@ impl ZkLoginPublicIdentifier {
 
 /// A JSON Web Key
 ///
-/// Struct that contains info for a JWK. A list of them for different kids can
+/// Struct that contains info for a JWK. A list of them for different keys can
```

Comment:
This seemed to be actually correct, https://www.googleapis.com/oauth2/v3/certs contains "kid" entries